src/System.Security.Cryptography.Pkcs/tests/CmsSignerTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using Xunit;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Security.Cryptography.Pkcs.Tests
+{
+    public class CmsSignerTests
+    {
+        [Fact]
+        public void SignCmsUsingAsymmetricSignatureFormatter()
```

Comment:
a) The method name no longer is appropriate.
b) I'd rather see this incorporated into the existing `SignedCmsTests`, to make sure that it covers RSA, DSA, and ECDSA as primary signer and counter-signer.
c) Negative tests